### PR TITLE
TO-1149: add USDT-A keeper

### PR DESCRIPTION
**Pull request by grandizzy**

### CODE REVIEW

File: flip/usdt-a/model-usdt.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  # share USDT_URL, DISCOUNT, and GASPRICE_MULTIPLIER
+  source ../env/environment.sh
+
+  # dynamic bid price
+  body=$(curl -s -X GET "$FLIP_USDT_URL" -H "accept: application/json")
+  usdtPrice=$(echo $body | jq '."tether".usd')
+  bidPrice=$(bc -l <<< "$usdtPrice * (1-$FLIP_USDT_DISCOUNT)")
+
+  echo "{\"price\": \"${bidPrice}\", \"gasPrice\": \"$(dynamic_gas)\"}"
```
Comment (EdNoepel) [inline flip/usdt-a/model-usdt.sh:16]: Models should not be providing gas price; this prevents the keeper from effectively managing the TX queue, replacing transactions as needed. I was surprised to see all the models are doing this.

**Review by EdNoepel — APPROVED** [review]
Though not relevant to the collateral addition, I'm concerned that model scripts for each collateral are specifying gas price.

File: env/environment.sh [hunk 2]
```diff
@@ -100,6 +100,15 @@ FLIP_MINIMUM_MANA_A_AUCTION_ID_TO_CHECK=0
 FLIP_MANA_URL="https://api.coingecko.com/api/v3/simple/price?ids=decentraland&vs_currencies=usd"
 FLIP_MANA_DISCOUNT=0.25 # e.g. 0.25 = 25% discount from FMV
 
+###### FLIP-USDT-A Config ######
```
Comment (godsflaw) [inline env/environment.sh:103]: We should update the `FIRST_BLOCK_TO_CHECK` to `10769102`.

File: flip/usdt-a/model-usdt.sh (hunk 1 above)
Comment (godsflaw) [reply flip/usdt-a/model-usdt.sh:16]: Yeah, we need another PR to remove the `gasPrice` and `dynamic_gas.sh`

**Review by godsflaw — CHANGES_REQUESTED** [review]
LGTM once the block comment is addressed.

TODO:
- remove dynamic_gas from the model.
- add PAXUSD-A for completeness.

File: env/environment.sh (hunk 2 above)
Comment (grandizzy) [reply env/environment.sh:103]: fixed in new commit

**Comment by grandizzy** [comment]
> LGTM once the block comment is addressed.
> 
> TODO:
> 
> * remove dynamic_gas from the model.
> * add PAXUSD-A for completeness.

I forgot that we add support for collateral with liq disabled, will send another PR in couple for it

File: env/environment.sh (hunk 2 above)
Comment (EdNoepel) [reply env/environment.sh:103]: Since the keeper isn't biting, this parameter isn't used for anything.  Recommend cleaning up in future change.

Comment (godsflaw) [reply env/environment.sh:103]: One can choose to run it in bite mode, I frequently test this mode to make sure we're not overwhelming a single node.  I would recommend leaving this param.

**Review by godsflaw — APPROVED** [review]

File: env/environment.sh (hunk 2 above)
Comment (EdNoepel) [reply env/environment.sh:103]: Didn't realize that.  I'm curious how many (hours?) it'd take to find ETH-A bite opportunities by scraping frobs from chain.

**Pull request merged**